app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -138,51 +157,48 @@ class SearchActivity : AppCompatActivity() {
             showHistory()
         }
 
-        searchEditText.addTextChangedListener(searchTextWatcher)
+        binding.searchEditText.addTextChangedListener(searchTextWatcher)
 
-        searchEditText.setOnEditorActionListener { _, actionId, _ ->
+        binding.searchEditText.setOnEditorActionListener { _, actionId, _ ->
             if (actionId == EditorInfo.IME_ACTION_DONE) {
-                findTrack(searchText)
+                findTrack()
                 true
             }
```

Comment:
🍏 true не используется, дальше все равно уходит false
```suggestion
            } else {
```

Reply:
Исправила